submissions/ogennaOgbogu/htmlForms/productPage.html

```diff
@@ -0,0 +1,158 @@
+<!DOCTYPE html>
```

Comment:
You are meant to continue working on the product page you worked on doing the html task

Reply:
Hi @adaugochi thank you, I've resolved it.